test/letters/containers/Main.unit.spec.jsx

```diff
@@ -1,55 +1,134 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
-import _ from 'lodash';
+import sinon from 'sinon';
 
 import { Main } from '../../../src/js/letters/containers/Main';
 
+const getBenefitSummaryOptionsSpy = sinon.spy();
+const getLetterListSpy = sinon.spy();
+const getMailingAddressSpy = sinon.spy();
+const getAddressCountriesSpy = sinon.spy();
+const getAddressStateSpy = sinon.spy();
+
 const defaultProps = {
+  letters: { },
+  destination: { },
   lettersAvailability: 'available',
-  letters: { }
+  benefitSummaryOptions: {
+    benefitInfo: '',
+    serviceInfo: '',
+  },
+  optionsAvailable: {},
+  getBenefitSummaryOptions: getBenefitSummaryOptionsSpy,
+  getLetterList: getLetterListSpy,
+  getMailingAddress: getMailingAddressSpy,
+  getAddressCountries: getAddressCountriesSpy,
+  getAddressStates: getAddressStateSpy
 };
 
+const testText = 'test child element';
+const children = (<span>{testText}</span>);
+
 describe('<Main>', () => {
-  it('should render', () => {
+  it('renders', () => {
     const tree = SkinDeep.shallowRender(<Main {...defaultProps}/>);
     const vdom = tree.getRenderOutput();
     expect(vdom).to.not.be.undefined;
   });
 
-  it('should not show loading spinner when letters are available', () => {
-    const props = _.merge({}, defaultProps, { lettersAvailability: 'available' });
+  it('renders its children when letters are available', () => {
+    const props = { ...defaultProps, children };
     const tree = SkinDeep.shallowRender(<Main {...props}/>);
-    expect(tree.subTree('LoadingIndicator')).to.be.false;
+    const childText = tree.subTree('span').text();
+    expect(childText).to.equal(testText);
   });
 
-  it('should show loading spinner when waiting for response', () => {
-    const props = _.merge({}, defaultProps, { lettersAvailability: 'awaitingResponse' });
+  it('shows a loading spinner when awaiting response', () => {
+    const props = { ...defaultProps, lettersAvailability: 'awaitingResponse' };
```

Comment:
Out of curiosity, why change these from `_.merge()`?

Reply:
Eh, wasn't explicitly necessary, I was just trying to remove unnecessary dependencies. Seemed like getting rid of lodash here was a good candidate since the prop object doesn't need to be nested and shallow clone functionality is easily replicable using native features. That being said, the prop object actually *is* nested, and while this nuance doesn't affect the current tests, I'd hate for it to break things down the road, so after thinking about this some more I'm changing it back to `lodash`. Oh well!

Reply:
Heh, I did check to see if it was going to break because of that shallow cloning nonsense, but it's golden. I don't think you have to worry about changing it back; if we have to have another test down the road that needs the default props deeply cloned, we can cross that bridge when we get there.